democracy/management/commands/democracy_remove_dupes.py

```diff
@@ -0,0 +1,41 @@
+from datetime import timedelta
+from optparse import make_option
+
+from django.core.management.base import BaseCommand
+from django.db.models import Count
+
+from democracy.models import SectionComment
+
+
+class Command(BaseCommand):
+    option_list = BaseCommand.option_list + (
+        make_option("--yes-i-know-what-im-doing", dest="nothing_can_go_wrong", action="store_true"),
+    )
+
+    def _remove_dupes(self, klass):
+        potential_dupes = klass.objects.values('content',).exclude(content='')\
```

Comment:
Here we cannot actually exclude empty content. Lots of identical plugin_data have been posted with empty content, and duplicate plugin_data should be removed as well.